insights/parsers/tests/test_spamassassin_channels.py

```diff
@@ -0,0 +1,55 @@
+import doctest
+import pytest
+
+from insights.parsers import spamassassin_channels
+from insights.parsers.spamassassin_channels import SpamassassinChannels
+from insights.tests import context_wrap
+
+DEFAULT = """
+/etc/mail/spamassassin/channel.d/sought.conf:CHANNELURL=sought.rules.yerp.org
+/etc/mail/spamassassin/channel.d/spamassassin-official.conf:CHANNELURL=updates.spamassassin.org
+""".strip()
+
+SPACES = """
+/etc/mail/spamassassin/channel.d/sought.conf:  CHANNELURL=sought.rules.yerp.org
+/etc/mail/spamassassin/channel.d/spamassassin-official.conf:CHANNELURL=updates.spamassassin.org
+""".strip()
+
+TWO_IN_FILE = """
+/etc/mail/spamassassin/channel.d/sought.conf:CHANNELURL=sought.rules.yerp.org
+/etc/mail/spamassassin/channel.d/sought.conf:CHANNELURL=sought2.rules.yerp.org
+/etc/mail/spamassassin/channel.d/spamassassin-official.conf:CHANNELURL=updates.spamassassin.org
+""".strip()
+
+
+TEST_CASES = [
+    (DEFAULT, {
+        "/etc/mail/spamassassin/channel.d/sought.conf": ["sought.rules.yerp.org"],
```

Comment:
I would use variables for both input and output to have it unified, e.g. (`DEFAULT`, `EXPECTED_DEFAULT`) etc.

Reply:
Good idea.